### PR TITLE
DDCE-959: Added Charity Regulator's warmup page

### PR DESCRIPTION
**Pull request by robert-webb-hmrc**
# DDCE-959

##New feature

Warmup page for regulator's page section.

## PR Suggestions
- Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?
- Have you done a visual check of the changes?
- Is there any still commented or unused code? Lingering printlns?
- Have things been implemented in a complicated way?
- Are the test still over the coverage threshold?
- Does the compiler throw a lot of warnings? 
- Have methods and tests been named clearly?
- Were there any changes in the config? Do changes need to be made in app-config-???

## Checklist PR Raiser
 - [x]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [x]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [x]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
 - [x]  I've squashed my commits - including the JIRA issue number in the commit message
 - [ ]  I've run a dependency check to ensure all dependencies are up to date

## Checklist PR Reviewer
 - [ ]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [ ]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [ ]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
 - [ ]  I've squashed my commits - including the JIRA issue number in the commit message
 - [ ]  I've run a dependency check to ensure all dependencies are up to date

### CODE REVIEW

File: app/views/regulatorsAndDocuments/StartCharityRegulatorView.scala.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+    @h1(messages("startCharityRegulator.heading"))
+
+    @p(Html(messages("startCharityRegulator.p1")))
+    @bullets("startCharityRegulator.b1", "startCharityRegulator.b2")
+    @p(Html(messages("startCharityRegulator.p2")))
+
+    @link(controllers.regulatorsAndDocuments.routes.IsCharityRegulatorController.onPageLoad(NormalMode).url, "site.continue",Some("linkButton"))
+
+}
```
Comment (prameya-phatak) [inline app/views/regulatorsAndDocuments/StartCharityRegulatorView.scala.html:40]: section name needs to be included in the title.Which is missing at the moment.

File: app/controllers/regulatorsAndDocuments/StartCharityRegulatorController.scala [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import repositories.UserAnswerRepository
+import views.html.regulatorsAndDocuments.StartCharityRegulatorView
+
+class StartCharityRegulatorController @Inject()(val userAnswerRepository: UserAnswerRepository,
+                                                identify: AuthIdentifierAction,
+                                                getData: UserDataRetrievalAction,
+                                                requireData: DataRequiredAction,
+                                                val controllerComponents: MessagesControllerComponents,
+                                                view: StartCharityRegulatorView
```
Comment (prameya-phatak) [inline app/controllers/regulatorsAndDocuments/StartCharityRegulatorController.scala:33]: Please change the alignment just like other controllers.

File: test/controllers/regulatorsAndDocuments/StartCharityRegulatorControllerSpec.scala [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+  override def beforeEach(): Unit = {
+    super.beforeEach()
+    reset(mockUserAnswerRepository)
+  }
+
+  private val view: StartCharityRegulatorView = injector.instanceOf[StartCharityRegulatorView]
+
+  private val controller: StartCharityRegulatorController = inject[StartCharityRegulatorController]
+
+  "CharityInformation Controller" must {
```
Comment (prameya-phatak) [inline test/controllers/regulatorsAndDocuments/StartCharityRegulatorControllerSpec.scala:52]: Please change the name to StartCharityRegulatorController

**Comment by robert-webb-hmrc** [comment]
Updated

**Pull request merged**